package.json

```diff
@@ -43,9 +43,9 @@
     "ember-cli-template-lint": "^1.0.0-beta.1",
     "ember-cli-uglify": "^2.1.0",
     "ember-code-snippet": "^2.4.0",
-    "ember-concurrency-async": "^0.2.0",
+    "ember-concurrency-async": "chancancode/ember-concurrency-async#encapsulated-task-v2",
     "ember-concurrency-decorators": "^2.0.0",
-    "ember-concurrency-ts": "^0.1.0",
+    "ember-concurrency-ts": "chancancode/ember-concurrency-ts#encapsulated-task-v2",
```

Comment:
🤷‍♂️ Not ideal, but these are development dependencies, and we have a chicken and egg problem. Either we merge it like this, once this is released as 1.2.1 I will release those and come back to update this. Or we merge this PR without the second commit, and I come back to update the Octane tests later.